introduction_to_amazon_algorithms/autogluon_tabular/Amazon_Tabular_Classification_AutoGluon.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<p>Branding nit: replace all occurrences of <code>Sagemaker</code> with <code>SageMaker</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Thanks for catching that! I have addressed for all notebooks in upcoming PR.</p>